modular/executor/execute_task.go

```diff
@@ -117,6 +118,40 @@ func (e *ExecuteModular) sealObject(ctx context.Context, task coretask.ObjectTas
 	return err
 }
 
+func (e *ExecuteModular) sealObjectV2(ctx context.Context, task coretask.ObjectTask, sealMsg *storagetypes.MsgSealObjectV2) error {
```

Comment:
can you add some comments to explain what is the differences between sealObject and sealObjectV2?